### ISSUE TITLE
Component example is not working if vuex property in config is undefined

### ISSUE THREAD

**Issue by happylittleboozer** [vue-styleguidist/vue-styleguidist#4]
Hi!
Thanks for the great tool.
In config documentation I found that vuex property is optional. I don`t use vuex and in my styleguide.config.js vuex property is not set, so all my examples from markdown files is not working.
But if I set it as path pointing to module that exports empty object, everything is working, like this:
```javascript
module.exports = {
  vuex: 'empty.js'
};
```
and in empty.js:
```javascript
export default {};
```
Can you fix this issue?

**Comment by rafaesc92** [comment]
Hi!

I solved the problem in v1.0.10, have you updated it?

**Comment by happylittleboozer** [comment]
Yes, problem is solved. Thanks!
